gen/dcos-config.yaml

```diff
@@ -367,22 +367,31 @@ package:
       }
 {% case "false" %}
 {% endswitch %}
-  - path: /etc/mesos-slave-dns.json 
+  - path: /etc/mesos-slave-dns.json
     content: |
       {
         "docker":
         [
             {
               "network_mode": "USER",
               "dns": {
-                "nameservers": [ 
+                "nameservers": [
                                 "198.51.100.1",
                                 "fd01:d::c633:6401"
                 ]
               }
             }
         ]
       }
+  - path: /etc/mesos-slave-image-gc-config.json
+    content: |
+      {
+        "image_disk_headroom": 0.1,
+        "image_disk_watch_interval": {
+          "nanoseconds": 300000000000
+        },
+        "excluded_images": []
```

Comment:
Does this need to be included if it's an empty array?
Or are you just suggesting where an admin could add images to exclude?

Reply:
It is an optional field and is not necessary to be included.

But I decided to pass in an empty array to let dc/os operators know there is such a field in container image gc config file and they can consume it if they did not read the image gc documents. Some operators might be very interested in using `excluded_images` (e.g., Uber's use cases).